### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by marcusn1301**
It is now possible to send applications, view all the applications on the admin page, filter by dates, and export as CSV.

The admin page can be accessed at /admin

### CODE REVIEW

File: backend/src/controllers/auth.ts [hunk 1]
```diff
@@ -16,13 +17,14 @@ body('phone').isString().isLength({min: 10, max: 15}).withMessage('Phone number
...
 body('password').isString().isLength({min: 8, max: 50}).withMessage('Password must be at least 8 characters long'),
+body('role').isString().isIn([ROLES.ADMIN, ROLES.USER]).withMessage('Role is invalid'),
 
 async (req: Request, res: Response, next: NextFunction) => {
     const errors = validationResult(req);
     if (!errors.isEmpty()) {
         return res.status(400).json({ errors: errors.array() });
     }
-    const { firstName, lastName, email, phone, country, dateBirth, password } = req.body;
+    const { firstName, lastName, email, phone, country, dateBirth, password, role } = req.body;
```
Comment (steffenhaeren) [inline backend/src/controllers/auth.ts:27]: Should firstName and lastName be merged to fullName here as well perhaps?

File: frontend/src/types/types.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
 export interface User {
+    _id: string;
     firstName: string;
```
Comment (steffenhaeren) [inline frontend/src/types/types.ts:3]: As well as here?

**Comment by steffenhaeren** [comment]
LGTM! Se kommentarer ang firstName og lastName

File: frontend/src/types/types.ts [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
     lastName: string;
```
Comment (steffenhaeren) [inline frontend/src/types/types.ts:4]: Here as well?

File: backend/src/controllers/auth.ts (hunk 1 above)
Comment (marcusn1301) [reply backend/src/controllers/auth.ts:27]: yeah sure

**Pull request merged**